integration-libs/opf/checkout/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnDestroy,
+  OnInit,
+  ViewContainerRef,
+} from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import {
+  GlobalFunctionsDomain,
+  OpfGlobalFunctionsFacade,
+} from '@spartacus/opf/base/root';
+import {
+  OpfPaymentMethodType,
+  PaymentPattern,
+  PaymentSessionData,
+} from '@spartacus/opf/checkout/root';
+import { Subscription } from 'rxjs';
+import { OpfCheckoutPaymentWrapperService } from './opf-checkout-payment-wrapper.service';
+
+@Component({
+  selector: 'cx-opf-checkout-payment-wrapper',
+  templateUrl: './opf-checkout-payment-wrapper.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class OpfCheckoutPaymentWrapperComponent implements OnInit, OnDestroy {
+  @Input() selectedPaymentId: number;
+
+  renderPaymentMethodEvent$ = this.service.getRenderPaymentMethodEvent();
+
+  RENDER_TYPES = OpfPaymentMethodType;
+
+  sub: Subscription = new Subscription();
+
+  constructor(
+    protected service: OpfCheckoutPaymentWrapperService,
+    protected sanitizer: DomSanitizer,
+    protected globalFunctionsService: OpfGlobalFunctionsFacade,
+    protected vcr: ViewContainerRef
+  ) {}
+
+  renderHtml(html: string): SafeHtml {
+    return this.sanitizer.bypassSecurityTrustHtml(html);
```

Comment:
Sonar flags this as a Security risk. 
This sort of bypass requires formal internal approval.